src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
 public class FileWork {
+
+    static final char STARTS_WITH = 'w';
+
     public String[] readFromFile(String fileName) {
-        return null;
+        String[] result;
+        StringBuilder allFile = new StringBuilder();
```

Comment:
let's improve the informativeness of the variable names

Reply:
renamed the variable